pkg/universe.dagger.io/alpha/kubernetes/helm/upgrade.cue

```diff
@@ -0,0 +1,131 @@
+package helm
+
+import (
+	"strings"
+	"list"
+
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+#Upgrade: {
+	// The image to use when running the action.
+	// Must contains the helm binary. Defaults to alpine/helm
+	image: *#Image.output | docker.#Image
```

Comment:
I'd like to see whatever we do around images be consistent across `upgrade.cue` and `build.cue`.

Could you also propose the same `image:` in build.cue? 🙏

Reply:
I have added the image field to the `install.cue` action, assuming this is what you meant by `build.cue`.